spec/features/movies/show_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+RSpec.describe 'As a user, I can click a movie title' do
+  describe 'and see movie details page' do
+    it "contains movie reviews cast and other details" do
+      OmniAuth.config.test_mode = true
+      OmniAuth.config.mock_auth[:google_oauth2] = OmniAuth::AuthHash.new(
+        Faker::Omniauth.google
+      )
+      visit root_path
+      click_button 'Log In with Google'
+
+      visit movies_path(search_terms: 'godfather')
```

Comment:
Had to search movie with 10 cast members and a review present. This was also more effective than navigating from top_40 so we could predict the path.